scss/bootstrap/overrides/style/_panel.scss

```diff
@@ -1,8 +1,9 @@
 .panel {
   @include box-shadow($crm-panels-box-shadow);
-
+  border-radius: $border-radius-small;
   border: none;
   margin-bottom: $grid-gutter-width;
+  overflow: hidden;
```

Comment:
I removed the overflow hidden and replaced it with another border-radius in .panel-body

plz check this commit
https://github.com/compucorp/org.civicrm.shoreditch/pull/5/commits/2f1e119b5635437b3866ff1717d11edc78c27fca